extensions/resource-deployment/src/test/azdataService.test.ts

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import 'mocha';
+import * as TypeMoq from 'typemoq';
+import assert = require('assert');
```

Comment:
`import assert as * from 'assert';`

But also any reason you're not using should? That's a much better framework for assertions and is what we typically are using for extensions.